tilequeue/command.py

```diff
@@ -577,7 +577,8 @@ def tilequeue_process(cfg, peripherals):
     assert n_simultaneous_query_sets > 0
     # reduce queue size when we're rendering metatiles to try and avoid the
     # geometry waiting to be processed from taking up all the RAM!
-    default_queue_buffer_size = max(1, 128 >> (2 * (cfg.metatile_size or 0)))
+    default_queue_buffer_size = max(
+        1, 128 >> ((2 * (cfg.metatile_size or 0))) + 1)
```

Comment:
Are you using bit shifting to represent a non-linear relationship? It'd be easier to read if it used Python's standard math syntax I think.